lib/dill/checkpoint.rb

```diff
@@ -11,7 +11,7 @@ class << self
       attr_accessor :rescuable_errors
     end
 
-    self.rescuable_errors = [StandardError]
+    self.rescuable_errors = [StandardError, Exception]
```

Comment:
I think we only need Exception here, since StandardError subclasses Exception. http://blog.nicksieger.com/articles/2006/09/06/rubys-exception-hierarchy/

I'm not sure we want to rescue Exception, though, given the other errors that subclass it. Maybe we should just add `RSpec::Expectations::ExpectationNotMetError` ?